pages/send/Send.tsx

```diff
@@ -197,17 +186,49 @@ export function Send() {
               </View>
             </TouchableWithoutFeedback>
           )}
-          {!isScanning && !keyboardOpen && (
+          {permissionStatus === PermissionStatus.UNDETERMINED && !keyboardOpen && (
             <>
               <View className="flex-1 h-full flex flex-col items-center justify-center gap-5">
-                <CameraIcon className="text-black w-32 h-32" />
-                <Text className="text-2xl">Camera Permissions Needed</Text>
+                <View className="flex flex-col gap-2 items-center">
+                  <CameraIcon className="text-foreground" size={72} />
+                  <Text className="text-2xl text-foreground">Camera Permissions Needed</Text>
+                </View>
```

Comment:
I think you're right because of the effect the user will be automatically prompted. We do not need the button below "Grant Permissions".

Can we also extract this into a component so we use the same one in Send and WalletConnection?

Reply:
I am just wondering whether we need this state at all? The user is asked for permission when the component initializes. If the user denies he will end up on the "Camera permissions denied" screen, if he confirms the camera will start up. :thinking: When would this screen even be displayed?

Reply:
sounds good to delete as much of that code as possible - no need to ask the user to grant permissions